spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -120,11 +130,48 @@ def resolution_center
         client.get_resolution_center(apple_id, platform)
       end
 
+      def platforms
+        platforms = []
+        version_sets.each do |version_set|
+          platforms << version_set.platform
+        end
+        platforms
+      end
+
+      def types
```

Comment:
Ah thought the comment was on platforms. Hmm seems like types is never used. Will remove